pyintelowl/cli/jobs.py

```diff
@@ -59,6 +59,7 @@ def jobs(ctx: ClickContext, id: int, all: bool, status: str):
 
 
 @jobs.command("poll", help="HTTP poll a currently running job's details")
+@click.option("--id", type=int, default=0, help="Poll job by ID")
```

Comment:
there should be no default value and mark it `required=True`. And change `help="HTTP poll a job for live updates"`